domains/DataCollection/Forms/IdentificationForm/index.js

```diff
@@ -1,39 +1,29 @@
-import React, { useState, useEffect } from 'react';
+import { Formik } from 'formik';
+import React, { useEffect, useState } from 'react';
 import {
   ActivityIndicator,
-  View, TouchableWithoutFeedback, Keyboard,
+  Keyboard,
+  TouchableWithoutFeedback, View,
 } from 'react-native';
-import { Formik } from 'formik';
-
-import { postIdentificationForm } from '../../../../modules/cached-resources';
-import { isEmpty } from '../../../../modules/utils';
-import { layout, theme } from '../../../../modules/theme';
-import I18n from '../../../../modules/i18n';
 
 import PaperButton from '../../../../components/Button';
+import ErrorPicker from '../../../../components/FormikFields/ErrorPicker';
 import PaperInputPicker from '../../../../components/FormikFields/PaperInputPicker';
 import yupValidationPicker from '../../../../components/FormikFields/YupValidation';
-import ErrorPicker from '../../../../components/FormikFields/ErrorPicker';
-import backgroundPostPatient from './utils';
+import { postIdentificationForm } from '../../../../modules/cached-resources';
+import I18n from '../../../../modules/i18n';
+import { layout, theme } from '../../../../modules/theme';
+import { isEmpty } from '../../../../modules/utils';
 import surveyingUserFailsafe from '../utils';
-
 import configArray from './config/config';
 
 const IdentificationForm = ({
   scrollViewScroll, setScrollViewScroll,
   setSelectedForm, setSurveyee, surveyingUser, surveyingOrganization
 }) => {
   useEffect(() => {
-    const interval = setInterval(() => {
```

Comment:
I removed the background posting patient because we're now using the push to post offline function